updates/v1.0.1/create_providers_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Winter\Storm\Database\Schema\Blueprint;
+use Winter\Storm\Database\Updates\Migration;
+use Winter\Storm\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('winter_sso_providers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->boolean('is_enabled')->nullable();
+            $table->string('name');
+            $table->string('slug')->index();
+            $table->string('client_id');
+            $table->string('client_secret')->nullable();
```

Comment:
The client credentials should be encrypted in the database

Reply:
Should we use the Hashable trait like the Auth User model ?

Reply:
Wouldn't it need to be the Encryptable trait so you could retrieve the values?

Reply:
Good point! Done.